t/perlcriticrc

```diff
@@ -1,41 +1,72 @@
 
-# $Id$
-
 severity = 5
-verbose  = 8
+verbose  = 3
 
 #------------------------------------------------------------------------------
 
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::Documentation::RequirePodSections
 [Documentation::RequirePodSections]
 lib_sections    = NAME|DESCRIPTION|SYNOPSIS|AUTHORS|COPYRIGHT
 script_sections = NAME|DESCRIPTION|SYNOPSIS|AUTHORS|COPYRIGHT
 
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::TestingAndDebugging::RequireUseWarnings
 [TestingAndDebugging::RequireUseWarnings]
+
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitMagicNumbers
 [ValuesAndExpressions::ProhibitMagicNumbers]
 
-#Might be to late to address
-[-Subroutines::ProhibitExplicitReturnUndef]
+# Might be to late to address, so it is disabled at various places in the code, could perhaps be 
+# revisited with a major release, since it would break backwards compatibility, the recommeded 
+# solution is bare return
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::Subroutines::ProhibitExplicitReturnUndef
+[Subroutines::ProhibitExplicitReturnUndef]
 
-#The violater of this is an overwrite for Class::Accessor
+# The violater of this is an overwrite for Class::Accessor
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::NamingConventions::ProhibitAmbiguousNames
 [-NamingConventions::ProhibitAmbiguousNames]
 
-#We are using Constants
+# We are using Constants
+# Perhaps exchange this for Readonly?
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitConstantPragma
 [-ValuesAndExpressions::ProhibitConstantPragma]
 
-#We currently utilize a package variable deliberately, should be addressed
-#again
+# We currently utilize a package variable deliberately, should be addressed again
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::Variables::ProhibitPackageVars
 [-Variables::ProhibitPackageVars]
 
-#For embedded SQL this is perfectly ok
+# For embedded SQL this is perfectly ok
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitImplicitNewlines
 [-ValuesAndExpressions::ProhibitImplicitNewlines]
 
-#Exchange for Module::Load?
+# Exchange for Module::Load?
```

Comment:
I'd propose `Module::Runtime` which seems to be the de-facto standard these days.

Reply:
Also, should we put that change in this PR too? It seems completely related and addressing one of the disabled policies...

Reply:
I have updated the comment, but pushed the decision to later,  I have no experience with Module::Runtime, but I patched Module::Load ages ago, so it has sort of stuck around with me. But in general I have no preference